src/PhpParser/NodeTraverser/RectorNodeTraverser.php

```diff
@@ -14,7 +14,9 @@ final class RectorNodeTraverser extends NodeTraverser
 {
     private bool $areNodeVisitorsPrepared = false;
 
-    /** @var PhpRectorInterface[]|NodeVisitor[] */
+    /**
+     * @var PhpRectorInterface[]|NodeVisitor[]
+     */
```

Comment:
This one need to be reverted as cause another bug that I reverted at main branch, I am removing STOP_TRAVERAL on direct refactor() return on downgrade rules.

Reply:
I will resolve the conflict.

Reply:
Fixed conflict.